commands/info.md

```diff
@@ -324,12 +328,22 @@ Here is the meaning of all fields in the **cpu** section:
 *   `used_cpu_user_main_thread`: User CPU consumed by the Redis server main thread
 
 The **commandstats** section provides statistics based on the command type,
-including the number of calls, the total CPU time consumed by these commands,
-and the average CPU consumed per command execution.
+ including the number of calls that reached command execution (not rejected),
+ the total CPU time consumed by these commands, the average CPU consumed
+ per command execution, the number of rejected calls
+ (errors prior command execution), and the number of failed calls
+ (errors within the command execution).
 
 For each command type, the following line is added:
 
-*   `cmdstat_XXX`: `calls=XXX,usec=XXX,usec_per_call=XXX`
+*   `cmdstat_XXX`: `calls=XXX,usec=XXX,usec_per_call=XXX,rejected_calls=XXX,failed_calls=XXX`
```

Comment:
@itamarhaber I know we don't currently document which info fields were added in each version, but maybe considering that this is a dict field we should? or maybe we should for all?

Reply:
@oranagra @itamarhaber should we add it as a note? example: Note: `rejected_calls` and `failed_calls` are available since XX

Reply:
Generally, we document the history, but in `INFO`'s case this has never been done. Better late than never? I don't think this applies in this case so we can merge as is.